packages/contracts/test/dollar/core/StakingShare.t.sol

```diff
@@ -304,18 +326,12 @@ contract RemoteDepositStateTest is DepositState {
         assertEq(stake1, stake2);
     }
 
-    function testHolderTokens_ShouldReturnHolderTokens() public {
-        ids.push(1);
-        uint256[] memory ids_ = stakingShare.holderTokens(stakingMinAccount);
-        assertEq(ids, ids_);
-    }
-
     function testSetUri_ShouldSetUri() public {
         string memory stringTest = "{'name':'Bonding Share','description':,"
         "'Ubiquity Bonding Share V2',"
         "'image': 'https://bafybeifibz4fhk4yag5reupmgh5cdbm2oladke4zfd7ldyw7avgipocpmy.ipfs.infura-ipfs.io/'}";
         vm.prank(admin);
-        stakingShare.setUri(stringTest);
+        stakingShare.setURI(stringTest);
```

Comment:
I think this should be `setUri`

Reply:
yes I made it so that it is consistent with OZ https://docs.openzeppelin.com/contracts/4.x/api/token/erc1155#ERC1155-_setURI-string-

Reply:
I understand why you would do that but I still think we should strive to maintain consistent code styles and I disagree with `NFT` being all capitalized in identifiers. I think that three letters or longer should be camel cased in identifiers.

Reply:
ok updated

Reply:
It looks like you left several still as `setURI` including this one.

Reply:
where exactly? AFAIK the only one remaining is         _setURI(newURI); cos it comes from OZ lib